harvester/utils.py

```diff
@@ -71,3 +74,30 @@ def put_object(self, body: str, key_name: str):
                 "ContentType": "application/json",
             }
         )
+
+
+class CFHandler:
+    def __init__(self, url: str = None, user: str = None, password: str = None):
+        self.target_endpoint = url if url is not None else os.getenv("CF_API_URL")
+        self.client = CloudFoundryClient(self.target_endpoint)
+        self.client.init_with_user_credentials(
+            user if user is not None else os.getenv("CF_USER"),
+            password if password is not None else os.getenv("CF_PASS"),
+        )
+
+        self.task_mgr = TaskManager(self.target_endpoint, self.client)
+
+    def start_task(self, app_guuid, command, task_id):
+        return self.task_mgr.create(app_guuid, command, task_id)
+
+    def stop_task(self, task_id):
+        return self.task_mgr.cancel(task_id)
+
+    def get_task(self, task_id):
+        return self.task_mgr.get(task_id)
+
+    def read_recent_app_logs(self, app_guuid, task_id=None):
+
+        app = self.client.v2.apps[app_guuid]
+        logs = filter(lambda lg: task_id in lg, [str(log) for log in app.recent_logs()])
+        return "\n".join(logs)
```

Comment:
Great start! Can we have a function that gets the current number of running tasks for a given app? This will be useful when managing how many tasks are running concurrently...

Reply:
added [here](https://github.com/GSA/datagov-harvesting-logic/pull/49/commits/9300f82460446d023ccefdf993cf32575624dbf7)

Reply:
went through this too quickly. the function i added grabs all the tasks of an app. i'll add another one which counts how many are running.